src/cmake/externalpackages.cmake

```diff
@@ -243,19 +243,18 @@ endif ()
 # Qt setup
 
 if (USE_QT)
+    set (qt5_modules Core Gui Widgets)
     if (USE_OPENGL)
-        set (QT_USE_QTOPENGL true)
+        list (APPEND qt5_modules OpenGL)
     endif ()
-    find_package (Qt4)
+    find_package (Qt5 COMPONENTS ${qt5_modules})
```

Comment:
Wait, maybe never mind. Builds and runs fine on Linux. Just not on my OSX laptop. Must be a problem on my end.